app/server.go

```diff
@@ -435,11 +426,17 @@ func (s *Server) Start() error {
 		handler = rateLimiter.RateLimitHandler(handler)
 	}
 
+	// Creating a logger for logging errors from http.Server at error level
```

Comment:
Can you please explain this further? What exactly is being set to mlog.LevelError? And why?

Reply:
`logger.StdLog` uses [zap.NewStdLog](https://github.com/uber-go/zap/blob/f4243df2af592a2e0ccdccb77e1ad44f78d735e5/global.go#L80) which creates a logger with info level.
Since we need a logger for error logging from http handlers, we need to use another function from zap called [zap.NewStdLogAt](https://github.com/uber-go/zap/blob/f4243df2af592a2e0ccdccb77e1ad44f78d735e5/global.go#L86), which lets us created a logger for a specific logging level.
Without this change, panic was logged at info level, which would be wrong and won't be logged at all if the logging level is set to error.

Reply:
K, got it, thanks for explaining.